### PR TITLE
Fix : HyperV VHD Copy issue

### PR DESCRIPTION
**Pull request by iamshital**
Fixed an issue where, LISAv2 was ignoring the VHD copy if it detected the existing disk identifier on remote host. But, if the VHD is updated, the disk identifier still  remains same, causing LISAv2 to not to use updated VHD.

### CODE REVIEW

**Comment by iamshital** [comment]
Logs from affected code: 
```
[oracle_6.5_rhck-ws2016] 03/20/2019 09:43:22 : [INFO ] Parent VHD path ****\oracle\oracle_6.5_rhck\oracle6.5.vhdx is on an SMB share.
[oracle_6.5_rhck-ws2016] 03/20/2019 09:43:23 : [INFO ] Checking if we have a local VHD with the same disk identifier on the host
[oracle_6.5_rhck-ws2016] 03/20/2019 09:43:23 : [INFO ] ****\oracle\oracle_6.5_rhck\oracle6.5.vhdx is already found at path \\localhost\f$\Hyper-V\VHDs\oracle6.5-688F8122C09141C7BBC60EDABD9DA135.vhdx
[oracle_6.5_rhck-ws2016] 03/20/2019 09:43:23 : [INFO ] Source and Destination VHD's 'LastWriteTimeUtc' property is different.
[oracle_6.5_rhck-ws2016] 03/20/2019 09:43:23 : [INFO ] ****\oracle\oracle_6.5_rhck\oracle6.5.vhdx will be copied at path \\localhost\f$\Hyper-V\VHDs\oracle6.5-688F8122C09141C7BBC60EDABD9DA135.vhdx
[oracle_6.5_rhck-ws2016] 03/20/2019 09:46:31 : [INFO ] Prerequiste: Prepare OS Disk VHDs_Destination_Path\LISAv2-OneVM-oracle_6.5_rhck-d-109-UI28-636886465971-role-0-diff-OSDisk.vhdx - Succeeded.```

File: Libraries/HyperV.psm1 [hunk 1]
```diff
@@ -348,11 +348,22 @@ function Create-HyperVGroupDeployment([string]$HyperVGroupName, $HyperVGroupXML,
             $newVhdName = "{0}-{1}{2}" -f @($vhdName, $infoParentOsVHD.DiskIdentifier.Replace("-", ""),$vhdSuffix)
             $localVHDPath = "{0}{1}{2}" -f @($hypervVHDLocalPath,[System.IO.Path]::DirectorySeparatorChar,$newVhdName)
             $localVHDUncPath = $localVHDPath -replace '^(.):', "\\${HyperVHost}\`$1$"
+            $CopyVHD = $false
             if ((Test-Path $localVHDUncPath)) {
                 Write-LogInfo "${parentOsVHDPath} is already found at path ${localVHDUncPath}"
+                $TargetVHD = Get-ChildItem -Path $localVHDUncPath
```
Comment (ader1990) [inline Libraries/HyperV.psm1]: here it should be Get-Item, why are you using child?

**Review by ader1990 — CHANGES_REQUESTED** [review]

File: Libraries/HyperV.psm1 (hunk 1 above)
Comment (iamshital) [reply Libraries/HyperV.psm1]: @ader1990 , no specific reason there. Get-Item / Get-ChildItem works same for the absolute filepath.

**Review by ader1990 — APPROVED** [review]

**Comment by ader1990** [comment]
All good from me ;)

**Pull request merged**